src/react-native.ts

```diff
@@ -0,0 +1,48 @@
+import path from 'path'
+
+import execa from 'execa'
+
+import log from './lib/log'
+import addScript from './lib/addScript'
+import copyDir from './lib/copyDir'
+
+const dependencies = ['react-native-localize']
```

Comment:
is this the only dependency? 🤩

Reply:
don't we need react-native-dotenv?

Reply:
Currently, it's just a recommendation and it's not being used in the code, what do you prefer?

Reply:
this is an opinionated CLI so I have no problem in including recommended dependencies as long as we're sure that it's a good, maintained, and helpful dependency.

but what about `react-native-dotenv`? Don't we need to have it in package.json (at least as a dev dep)?

Reply:
Just my two cents, but in my opinion `react-native-config` works way better with envs than `react-native-dotenv`. I had that experience with my current project.